src/commit.ml

```diff
@@ -0,0 +1,5 @@
+(* : commit.ml,v 7.00 2017-06-05 07:59:58 hg Exp $ *)
+(* Copyright (c) 1998-2017 INRIA *)
+
+value nbr = "e741073";
+value date =  "Mon Jun 5 07:21:04 2017 +0200";
```

Comment:
extra space and this file should not be put under git (same as compilation.ml) because it changes all the time